cost/scheduled_reports/scheduled_report.pt

```diff
@@ -535,7 +535,7 @@ script "js_generate_report", type: "javascript" do
     }
 
     var chartData = "chd=a:" + chartDataArray.join('|')
-    var encodedCategories = encodeURIComponent(categories.join('|'))//.replace(/%7C/gi,'|');
+    var encodedCategories = encodeURIComponent(categories.join('|')).replace(/[(]/gi,'%28').replace(/[)]/gi,'%29');
```

Comment:
Question - with this change not only have you encoded (), but you've undone something that looks like it unencoded | . was that intentional? and will it be ok?

Reply:
If you look carefully, the unencoding of `|` was actually commented out. I missed that when cleaning up my debug code during the initial PR of the chart enhancement.